src/test/ui/rfc-2497-if-let-chains/syntax-ambiguity-2015.rs

```diff
@@ -1,38 +0,0 @@
-// edition:2015
```

Comment:
these are not entirely redundant. They are testing on multiple editions

Reply:
Per Discord conversation: The code is edition independent (and was before). We seemingly don't have an easy way to test this for both editions in a convenient and non-duplicating way. We also don't typically test everything for both editions cause that'd inflate testing time notably.